.gitattributes

```diff
@@ -0,0 +1,16 @@
+* text=auto
```

Comment:
I don't like adding this file unless absolutely necessary. It exists in the .NET client due to the fact that when the Mono build was added, line-endings caused issues (may not be an issue now).

* The Ruby client is one of the oldest client libs Basho supports, and line-endings have never been an issue. I searched issues on GH for "line ending" and get no hits.
* Ruby devs can be assumed to be "in the know" when it comes to git and line endings, and should have git configured appropriately for their environment.
* I develop for the Ruby client on both Linux and Windows, and don't have line-ending issues when I follow [GitHub's recommendations](https://help.github.com/articles/dealing-with-line-endings/) on both platforms.